src/queries/SUE/areaService.tsx

```diff
@@ -1,10 +1,13 @@
 import { storageHelper } from '../../helpers';
 
 export const areaService = async () => {
-  const globalSettings = await storageHelper.globalSettings();
-  const { apiConfig = {} } = globalSettings?.settings?.sue || {};
+  const configurations = await storageHelper.configurations();
+  const { sueConfig = {} } = configurations;
+  const { apiConfig = {}, geoMap = {} } = sueConfig;
 
-  const { apiKey, serverUrl, id } = apiConfig.areaService || {};
+  const { apiKey, serverUrl } = apiConfig.areaService || {};
+
+  const id = geoMap.areas[0].id || apiConfig.areaService.id;
```

Comment:
do you think we could need some `?` to secure the objects accesses?

Reply:
Yes, that's right. I forgot about this safety. 👍